ui/src/app/search/rich-skill-search-results.component.spec.ts

```diff
@@ -1,27 +1,24 @@
-import { HttpClientTestingModule } from "@angular/common/http/testing"
-import { Type } from "@angular/core"
-import { async, ComponentFixture, TestBed } from "@angular/core/testing"
-import { Title } from "@angular/platform-browser"
-import { ActivatedRoute, Router } from "@angular/router"
-import { ActivatedRouteStubSpec } from "test/util/activated-route-stub.spec"
-import { createMockPaginatedSkills, createMockSkillSummary } from "../../../test/resource/mock-data"
-import {
-  AuthServiceStub,
-  RichSkillServiceStub,
-  SearchServiceData,
-  SearchServiceStub
-} from "../../../test/resource/mock-stubs"
-import { AppConfig } from "../app.config"
-import { EnvironmentService } from "../core/environment.service"
-import { PublishStatus } from "../PublishStatus"
-import { ApiAdvancedSearch, ApiSearch } from "../richskill/service/rich-skill-search.service"
-import { RichSkillService } from "../richskill/service/rich-skill.service"
-import { TableActionDefinition } from "../table/skills-library-table/has-action-definitions"
-import { ToastService } from "../toast/toast.service"
-import { RichSkillSearchResultsComponent } from "./rich-skill-search-results.component"
-import { SearchService } from "./search.service"
+/* tslint:disable:no-string-literal */
```

Comment:
Let's back these import changes for now? And move towards a strong editorconfig and maybe prettify?

Reply:
Now the imports are as before

Reply:
Do we still want the tslint hint? Those check add value, so we should only disable if needed.